stdlib/_ctypes.pyi

```diff
@@ -44,6 +44,8 @@ if sys.platform == "win32":
     def FormatError(code: int = ...) -> str: ...
     def get_last_error() -> int: ...
     def set_last_error(value: int) -> int: ...
+    def LoadLibrary(__name: str, __load_flags: int) -> int: ...
```

Comment:
```suggestion
    def LoadLibrary(__name: str, __load_flags: int = 0) -> int: ...
```

Reply:
Can confirm from local testing on a Windows machine.

Reply:
Yes, it does have `int load_flags = 0;`